fire/cli/mtl.py

```diff
@@ -254,17 +253,12 @@ def importer_observationer(projektnavn: str) -> pd.DataFrame:
         except NoResultFound:
             fire.cli.print(f"Ukendt punkt: '{punktnavn}'", fg="red", bg="white")
             sys.exit(1)
+        kanonisk_ident[punktnavn] = ident
 
-        if ident != punktnavn:
-            kanonisk_ident[punktnavn] = ident
-
-    for ident, kanon in kanonisk_ident.items():
-        hvor = [idx for idx, val in enumerate(fra) if val == ident]
-        for i in hvor:
-            fra[i] = kanon
-        hvor = [idx for idx, val in enumerate(til) if val == ident]
-        for i in hvor:
-            til[i] = kanon
+    for i in range(len(fra)):
+        fra[i] = kanonisk_ident[fra[i]]
```

Comment:
Du bør kunne bruge en list comprehension her:

```suggestion
fra = [kanonisk_ident[ident] for ident in fra]
```

Det er mere Pythonsk og næsten med sikkerhed hurtigere (ikke at det betyder noget her).

Reply:
Tak! Kunne ikke få det til at virke igår, da jeg forsøgte med en tuple comprehension:
```python
fra = (kanonisk_ident[ident] for ident in fra)
```
men det  udtryk returnerer bare et generatorobjekt, hvilket er fint hvis det er det man har brug for - men her skulle jeg naturligvis ("naturligvis" set i morgenens klare lys) bruge:
```python
fra = tuple(kanonisk_ident[ident] for ident in fra)
```
Har også rettet de to oprindelige `list`er `fra`og `til` til `tuple`r: Der er ikke brug for mutationer nogetsteds i deres korte levetid.

Har derudover rettet endnu en overset suboptimalitet: Undgå at slå alle punkter op to gange. Den rettelse er meget tæt på at halvere køretiden.